docs/Contributing/Testing-and-local-development.md

```diff
@@ -483,7 +483,7 @@ Once you have access to ABM, follow [these guided instructions](../Using-Fleet/M
 
 The server also needs a private key + certificate to identify with Apple's [APNs](https://github.com/fleetdm/fleet/blob/main/tools/mdm/apple/glossary-and-protocols.md#apns-apple-push-notification-service) servers, and another for [SCEP](https://github.com/fleetdm/fleet/blob/main/tools/mdm/apple/glossary-and-protocols.md#scep-simple-certificate-enrollment-protocol).
 
-To generate both, follow [these guided instructions](../Using-Fleet/Mobile-device-management.md#apple-push-notification-service-apns).
+To generate both, follow [these guided instructions](../Using-Fleet/Mobile-device-management.md#apple-push-notification-service-ap-ns).
 
```

Comment:
https://github.com/fleetdm/fleet/blob/main/docs/Using-Fleet/Mobile-device-management.md#apple-push-notification-service-apns

I don't think this link is broken or the fix is correct?

Reply:
@RachelElysia I think `ap-ns` is right. For some reason, the fleetdm.com website expects the link to be https://fleetdm.com/docs/using-fleet/mobile-device-management#apple-push-notification-service-ap-ns.

Here's a Loom recording that shows this: https://www.loom.com/spaces/All-Fleet-67132/folders/historicalmisc-86bc8a2d884c4313ac3a4da785510ccd

Reply:
You're right @noahtalerman , this is such a weird discrepancy between github and our website. If you were to click the anchor on github, it won't direct to the right place unless its apns without a hypen, but on our website it won't direct to the right place unless its ap-ns with a hyphen

Reply:
> If you were to click the anchor on github, it won't direct to the right place

Shoot that's unfortunate...

@RachelElysia what do you think about merging this PR so we can move quickly? We expect users to use docs on fleetdm.com/docs and not GitHub. So I think we can prioritize making the fix on the website.

@eashaw can weigh in at any time after we merge in the fix.

Reply:
>@eashaw do you know why the website expects ap-ns in the anchor link and not apns? (no hyphen) I think I would expect apns to work.

This is because of how we're generating IDs for headings in Markdown content. We're currently using Lodash's `.kebabCase()` method to convert the heading text to create the headings ID, which separates changes in letter casing with hyphens.

Reply:
> separates changes in letter casing with hyphens

@eashaw hmm, wouldn't this mean the converted text for `APNs` would be `apn-s` instead of `ap-ns`?

Reply:
@noahtalerman that happens because the `_.kebabCase()` method also converts camel case strings into kebab case. 

We should be able to change this behavior by converting the heading text to lowercase before creating the kebab case IDs. I'll make an issue for updating this and add it to the CX backlog.

Reply:
> the _.kebabCase() method also converts camel case strings into kebab case

@eashaw ah! Ok. Thanks for the clarification.

Re: Adding to the backlog: Sounds good!